client/src/components/PatientProfile.component.js

```diff
@@ -1,67 +1,148 @@
 import React, {Component} from 'react';
-import Navbar from "./Nbar.component";
 import { Link } from 'react-router-dom';
 import axios from 'axios';
 
-//assigned doctor
-//symptoms delete to be change for update
-const PatientEntry = props => (
-  <tr>
-    <td>{props.patientEntry.patientName}</td>
-    <td>{props.patientEntry.form}</td>
-    <td>{props.patientEntry.additionalNote}</td>
-    <td>{props.patientEntry.doctorNote}</td>
-    <td>{props.patientEntry.immediateAttention}</td>
-    <td>{props.patientEntry.entryDate}</td>
-    <td>
-      <Link to={"/edit/"+props.patientEntry._id}>edit</Link> | <a href="#" onClick={() => { props.deletePatientEntry(props.patientEntry._id) }}>delete</a>
-    </td>
-  </tr>
-)
-
-class PatientProfile extends Component{  
-    constructor(props) {
+export default class PatientProfile extends Component{ 
+   constructor(props){
     super(props);
-        this.state = {patientEntries: []};
+    this.state = {
+        symptom1:'',
+        patientFullName:'',
+        firstName:'',
+        lastName:'',
+        assignedDoctor:'',
+        age:'',
+        email:'',
+        patient:[],
+        patientEntry: [],
+        patientname:'',
+        patientId:'',
+        form:[],
+        date:''
     }
-     componentDidMount() {
-    axios.get('http://localhost:5000/patient/:id/profile')
-      .then(response => {
-        this.setState({ patientEntries: response.data })
+   }
+    
+componentDidMount(){
+     axios.get('http://localhost:5000/patient/:id')
```

Comment:
instead of using '/:id' we need to use actual ID passed from Ryan's login, I still don't know how we are doing that